prow/pjutil/pjutil.go

```diff
@@ -221,6 +223,7 @@ func PeriodicSpec(p config.Periodic) kube.ProwJobSpec {
 func BatchSpec(p config.Presubmit, refs kube.Refs) kube.ProwJobSpec {
```

Comment:
Need to update PeriodicSpec as well.

Reply:
There is no implied top level ref for periodic, so there is no top level field. The bool is part of the refs struct so it's in extra_refs for free